ContourTree/ContourTree.cpp

```diff
@@ -145,6 +145,7 @@ ContourTree::computeArcMap() {
 
     return {arcNo, nodeids, nodefns, nodeTypes, arcs};
 }
+}
```

Comment:
Was this added because of the additional loop? Based in this, the return statement is anyway inside the second loop.  Was the first loop a copy paste error?